### DISCUSSION TITLE
Selective Verse searching and cross chapter ranges

### DISCUSSION THREAD

**Discussion by AndyPandy1189** [tim-hub/obsidian-bible-reference#218] [Ideas]
I love this plug-in! I use it every day for my daily readings so I can take notes with them. A couple of ideas for pulling in the verses:

### Selective Verse searching
In some of my daily readings that I get, there are sometimes some gaps between verses that I would love to be able to enter 1 command for to handle. Currenty, I have to enter each one.

For example, PS 119:14, 24, 72, 103, 111, 131. 

This is used in responsorial Psalms a lot. Currently I have to enter this as separate entries (--ps 119:24; --ps 119:72, etc.). If I could enter this as one (--ps 119:14,24,72,103,111,131) to pull in those specific verses, that would be ideal.

### Cross chapter Ranges
When a verse range bridges multiple chapters in a book, then there is no really a way for me to pull that in other than separate entries. Currently I would have to enter --Ez 2:8-[Figure our the last verse in the chapter] --Ez 3:[Enter 1 for the first verse in the chapter]-4. Instead I would ideally enter --Ez 2:8--3:4 Or something like this.

**Comment by MannaBytes** [comment]
Cross-chapter Ranges would be an excellent addition. I find myself wanting to do things like this all the time, just like the OP:
--Ez 2:8--3:4
